.ci/docker/test/verify.sh

```diff
@@ -51,21 +51,16 @@ if [ -z "${BASE_IMAGE}" ]; then
   exit 1
 fi
 
-# Check that the base image exists
-if ! baseImageInfo="$(docker manifest inspect "${BASE_IMAGE}")"; then
-  echo >&2 "No known Docker base image ${BASE_IMAGE} exists; did you pass the right name?"
-  exit 1
+DIGEST_REGEX="BASE_DIGEST=\"(sha256\:[a-f0-9\:]+)\""
+DOCKERFILE=$(<"${BASH_SOURCE%/*}/../../operate.Dockerfile")
+if [[ $DOCKERFILE =~ $DIGEST_REGEX ]]; then
+    DIGEST="${BASH_REMATCH[1]}"
+    echo "Digest found: $DIGEST"
+else
+    echo >&2 "Docker image digest can not be found in the Dockerfile"
+    exit 1
 fi
 
```

Comment:
I think a check like https://github.com/camunda/zeebe/blob/82d1f59863089d8fc82c4fd9e69a5f2a3d396c48/docker/test/verify.sh#L47-L57 should be introduced against `$ARCHITECTURE` in this script - because that is currently not validated anymore.

Reply:
Ah cool!